src/settings/mod.rs

```diff
@@ -325,16 +330,38 @@ fn resolve_codes<'a>(specs: impl IntoIterator<Item = RuleCodeSpec<'a>>) -> FxHas
                 if selector.specificity() == specificity {
                     rules.extend(selector);
                 }
+
+                if let RuleSelector::Prefix {
+                    prefix,
+                    redirected_from: Some(redirect_from),
+                } = selector
+                {
+                    redirects.insert(redirect_from, prefix);
+                }
             }
             for selector in spec.ignore {
                 if selector.specificity() == specificity {
                     for rule in selector {
                         rules.remove(&rule);
                     }
                 }
+
+                if let RuleSelector::Prefix {
+                    prefix,
+                    redirected_from: Some(redirect_from),
+                } = selector
+                {
+                    redirects.insert(redirect_from, prefix);
+                }
             }
         }
     }
+
+    for (from, target) in redirects {
+        // TODO(martin): This belongs into the ruff_cli crate.
+        crate::warn_user!("`{from}` has been remapped to `{}`", target.as_ref());
```

Comment:
This will still warn twice if you specify an invalid code in both `select` and `fixable`, right? I'm ok with that, but I want to make sure I understand what's going on. (We could lift the redirect map up a level, or something, but seems annoying.)